### DISCUSSION TITLE
HEIC images as a source for Next/Image

### DISCUSSION THREAD

**Discussion by chrskerr** [vercel/next.js#30043] [Help]
Hello!

Is it possible to enable support for HEIC images as an input type for Next/Image? 

I have an app that lets users upload images directly, and I was hoping to write as little post-processing code as possible :) 

Edit: I just had a little dig into the code, and it appears to be using Sharp underneath which (from memory) needs a custom built libvips to load heic... so not looking good :(

Thanks!
Chris

**Comment by relief** [comment]
+1 to it, want heic support badly

**Comment by hungHatto** [comment]
is there any solution for this ?
i tried [https://stackoverflow.com/questions/74856178/how-to-convert-heic-to-jpg-in-react](this) but not work as same as the example.

**Comment by clipartinc** [comment]
This has worked for me...
https://www.npmjs.com/package/heic-convert
